front/lib/api/assistant/action.ts

```diff
@@ -0,0 +1,30 @@
+import tracer from "dd-trace";
```

Comment:
I moved this to a new file from server_side_props_helper. Since we will fetch all of them on the client side, I wonder if we might be able to build actions on the client side too 🤔